5.5/examples/replica/mysql_replica.json

```diff
@@ -158,11 +158,11 @@
                 "env": [
                   {
                     "name": "MYSQL_MASTER_USER",
```

Comment:
It doesn't, really. I thought `MYSQL_REPLICATION_USER` is clearer than `MYSQL_MASTER_USER`, but then stepped back on changing the env var. What I wanted is to hear a second opinion.